third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py

```diff
@@ -9,8 +9,10 @@
 def _get_fk_table_name(column):
 	for fk in column.foreign_keys:
 		colspec = fk._get_colspec()
-		tablename, _ = colspec.split(".")
-		return tablename
+		splitted = colspec.split(".")
+		assert len(splitted) >= 2
+		# second to the last is the table name.
+		return splitted[-2]
```

Comment:
i think remove comment and do this instead
```suggestion
		tablename = splitted[-2]
		return tablename
```

Reply:
I have updated this, incorporating some of your suggestion.  I will land this now, but if you still have concerns, I'm happy to revise.